packages/frontend/src/components/TestSubstep/index.tsx

```diff
@@ -43,11 +51,7 @@ function serializeErrors(graphQLErrors: any) {
     try {
       return {
         ...error,
-        message: (
-          <pre style={{ margin: 0, whiteSpace: 'pre-wrap' }}>
-            {JSON.stringify(JSON.parse(error.message as string), null, 2)}
-          </pre>
-        ),
+        errorDetails: JSON.parse(error.message as string),
```

Comment:
comment: out of scope for this PR, but I just found out about [apollo error extensions](https://github.com/apollographql/apollo-server/pull/3408). We might be able to leverage that to get rid of our jank JSON.parse in `serializeErrors`